packages/smart-wallet/test/supports.js

```diff
@@ -117,13 +117,15 @@ export const makeMockTestSpace = async log => {
   produce.zoe.resolve(zoe);
   produce.feeMintAccess.resolve(feeMintAccessP);
 
-  const vatLoader = name => {
+  /** @type {VatLoader} */
```

Comment:
a little loose since the type says it can load any vat and this handles just two of them

Reply:
I ended up restricting the loadable `name`s.